### ISSUE TITLE
Midpoint Review - Accessibility Feedback - MHV Secure Messaging, Landing Page/Navigation Updates

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#98225]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Lots of good stuff here, I really like the direction you've gone!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Must:** The color contrast between the IPE background and the white page background is pretty low. That's not so much a problem for the main content of the IPE, but the arrow pointing to its related element isn't likely to be seen by some low vision users (or someone outside on a sunny day, or someone with an old monitor like mine, etc). I'd recommend exploring other colors, border effects, etc.
- [ ] **Must:** As this is built out, make sure that the arrow is coded such that it always points to what it's supposed to point to, at all browser zoom levels. WCAG requires support up to 400%, but the key there is "up to" --- meaning, 100%, 150%, 200%, 250%, etc. I don't think you need to validate every single individual percentile, but it's a CSS challenge to make sure everything stays lined up for each instance of an IPE.
- [ ] **Must:** There's a lot of potential for ambiguity in "Don't show again," especially for screen reader users who don't get the visual cues of the IPE. Am I not showing the IPE message, or the element that's being pointed to, or something else? I think there's a content solution here, something like "Don't show this tip again" or similar. There was a lot of discussion of alternative phrasings for the button, and I'm fine with pretty much anything so long as the object of the action is clear.
- [ ] **Should:** I like what you're planning with `aria-describedby` associating the IPE message with the thing it's talking about. I'm a little less certain about the idea of having part but not all of the IPE message associated via ARIA. As you think about guidance in the design system, I'd definitely lean more in the direction of "make the whole thing associated via ARIA" and have breaking it up be special, clearly-articulated situations where that makes sense. I think the example you shared makes sense, but I don't know what might come from other teams picking up the component in the future.

## Governance team actions
- [ ] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [ ] Add the touchpoint labels
- [ ] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
